baseplate/__init__.py

```diff
@@ -124,8 +124,9 @@ def new(cls) -> "TraceInfo":
         with any upstream requests.
 
         """
-        trace_id = str(random.getrandbits(64))
-        return cls(trace_id=trace_id, parent_id=None, span_id=trace_id, sampled=None, flags=None)
+        trace_id = "{0:0{1}x}".format(random.getrandbits(128), 32)
```

Comment:
padded string with leading 0's to reach 32 characters - necessary in case hex string comes up short.

Reply:
```suggestion
        trace_id = f"{random.getrandbits(128):032x}"
```

Parameterizing the 32 makes it quite hard to read the format string,  and using f-strings makes this a little easier to understand as well.